redirects/linux.nginx

```diff
@@ -0,0 +1,344 @@
+# Top level redirects from NuDocs 1 to NuDocs 2
+# =============================================
+
+location /minio/baremetal/introduction/minio-overview.html {
+     return 301 https://min.io/minio/docs/linux/operations/concepts.html?ref=docs-redirect;
```

Comment:
Looks like many of these links have the URL folders swapped.
Replace `min.io/minio/docs` with `min.io/docs/minio` throughout.